chore: Link to child pages in SVG, URI section (#39470)
mdn/content: files/en-us/web/uri/index.md

## Reference

The [URI reference](/en-US/docs/Web/URI/Reference) provides details about the components that make up a URI.

- [URI schemes](/en-US/docs/Web/URI/Reference/Schemes)
  - : The first part of the URI, before the `:` character, which indicates the protocol the browser must use to fetch the resource.
- [URI authority](/en-US/docs/Web/URI/Reference/Authority)
  - : The section that comes after the scheme and before the path.
    It may have up to three parts: `user` information, `host`, and `port`.
- [URI fragment](/en-US/docs/Web/URI/Reference/Fragment)
  - : An optional part at the end of a URI starting with a `#` character.
    It is used to identify a specific part of the resource, such as a section of a document or a position in a video.

## Guides

The [URI guides](/en-US/docs/Web/URI/Guides) help you work with URIs on the web.

- [Choosing between www and non-www URLs](Choosing_between_www_and_non-www_URLs)
  - : Guidance on when sites should use a `www.` prefix in URLs (`www.example.com` vs `example.com`).

> ## URLs and URNs

Reviewer: Perhaps move this thing to "reference"? It's supposed to be a list of links.

Author: There were many side effects of moving this, see 54b4e6e19d45546d06ff4f60af9e6f98ca2e8375

>   - : A recurring question among website owners is whether to choose 'www' (`www.example.com`) or 'non-www' (`example.com`) URLs for your projects.

Reviewer 2: FWIW I bet its actually a very rare question :-)

```suggestion
  - : Guidance on when sites should prefer 'www' (`www.example.com`) or 'non-www' (`example.com`) URLs.
```

Author: Good point. I've included the suggestion from Josh above, too https://github.com/mdn/content/pull/39470#pullrequestreview-2830638443

>   - : A recurring question among website owners is whether to choose 'www' (`www.example.com`) or 'non-www' (`example.com`) URLs for your projects.

Author: >Why single quotes?

Updated in https://github.com/mdn/content/pull/39470/commits/b16ae2acd1da83279f594068a9577d5e4f60b95b which I think is a bit clearer.

>this is why I don't like the current IA. The sections below are supposed to be the list of links, but organized in a more organic way, rather than as an uninteresting, unordered list. The URI docs are supposed to be cut by the components, just like API docs are primarily cut by APIs, not by reference/guide.

No I appreciate sharing - those are rough edges -> these changes in particular highlight when a landing page for a technology/topic lists everything (all guides & references), then a guide / reference page duplicates a subset of that, it's not so elegant, I agree. At the least, the reference and guide pages should be exhaustive.

For cutting by URI component, do you mean it would be better to have:

```
./files/en-us/web/uri
├── authority
│   ├── guides
│   └── reference
├── fragment
│   ├── …
│   └── reference
└── schemes
    ├── …
    └── reference
```

v

```
./files/en-us/web/uri
├── guides
│   └── choosing_between_www_and_non-www_urls
└── reference
    ├── authority
    ├── fragment
    │   └── text_fragments
    └── schemes
        ├── data
        ├── javascript
        └── resource
```

https://github.com/mdn/content/tree/main/files/en-us/web/uri/reference

>   - : A recurring question among website owners is whether to choose 'www' (`www.example.com`) or 'non-www' (`example.com`) URLs for your projects.

Reviewer: > For cutting by URI component, do you mean it would be better to have:

Yes that's what I meant.

>   - : A recurring question among website owners is whether to choose 'www' (`www.example.com`) or 'non-www' (`example.com`) URLs for your projects.

Author: I guess we could do - currently the section is a little light on content so it's hard to see much of a different, but this is not a bad way to organize it, indeed.

>   - : A recurring question among website owners is whether to choose 'www' (`www.example.com`) or 'non-www' (`example.com`) URLs for your projects.

Reviewer: I don't think it's productive to change everything again—I can live with the current organization. I think if the other sections on the main landing page (URI vs URN & URI structure) get moved it would look a lot cleaner.

>   - : A recurring question among website owners is whether to choose 'www' (`www.example.com`) or 'non-www' (`example.com`) URLs for your projects.

Author: Let's revisit this

>(URI vs URN & URI structure) get moved it would look a lot cleaner

Definitely, these are some leftovers that we can tidy and flesh out properly.
